src/main/java/io/appform/dropwizard/actors/base/UnmanagedConsumer.java

```diff
@@ -59,14 +59,17 @@ public UnmanagedConsumer(final String name,
         this.queueName = NamingUtils.queueName(config.getPrefix(), name);
         this.retryStrategy = retryStrategyFactory.create(config.getRetryConfig());
         this.exceptionHandler = exceptionHandlingFactory.create(config.getExceptionHandlerConfig());
+        this.messageMetadataProvider = new MessageMetadataProvider(getMessageMetaGeneratorClasses(config));
```

Comment:
This makes testing difficult. Do not create objects using constructors in other constructors if possible